pages/lists/[listId]/activity.tsx

```diff
@@ -125,7 +126,7 @@ const ListActivityPage = ({ list, numberOfContributors, isError, activityData }:
       {isError ? (
         <Error errorMessage="Unable to load list activity" />
       ) : (
-        <div className="lg:grid lg:grid-cols-2 lg:grid-rows-2 gap-4 flex flex-col">
+        <div className="flex flex-col gap-4 lg:grid lg:grid-cols-2 lg:grid-rows-2">
```

Comment:
Refactor (blocking): This isn't eslint rearranging these. There are also other changes like this below.

Reply:
Its a vscode extension i have enabled... i've disabled for this project👍